server/rooms.ts

```diff
@@ -1447,6 +1447,12 @@ export class GameRoom extends BasicChatRoom {
 
 		this.tour = options.tour || null;
 		this.parent = options.parent || (this.tour && this.tour.room) || null;
+		// copy mutes from parent room
```

Comment:
I would rather inherit them (search parents for mutes when checking for mutes) than copy them. That way, new mutes would also apply to subrooms.

Reply:
.....yeah, that's a much better way to do this